src/care/config.h.in

```diff
@@ -25,6 +25,7 @@
 #cmakedefine CARE_ENABLE_FUSER_BIN_32
 #cmakedefine01 CARE_ENABLE_PARALLEL_LOOP_BACKWARDS
 #cmakedefine01 CARE_ENABLE_PINNED_MEMORY_FOR_SCANS
+#cmakedefine CARE_GPU_MEMORY_IS_ACCESSIBLE_ON_CPU
```

Comment:
Is this something you expect the user to set? An option in the top level CMakeLists.txt that defaults to OFF would be good.